letsencrypt/client.py

```diff
@@ -268,20 +268,22 @@ def save_certificate(self, certr, chain_cert, cert_path, chain_path):
         :param list chain_cert:
         :param str cert_path: Candidate path to a certificate.
         :param str chain_path: Candidate path to a certificate chain.
+        :param str fullchain_path: Candidate path to a full cert chain.
 
-        :returns: cert_path, chain_path (absolute paths to the actual files)
+        :returns: cert_path, chain_path, fullchain_path (absolute paths to the actual files)
```

Comment:
Long line. You could break or remove/shorten the comment about paths being absolute.